### PR TITLE
feat: Enforce non-negativity of Rabi rotations

### PR DESCRIPTION
**Pull request by tiborr**
Changes proposed in this pull request:

- Enforce non-negativity of Rabi rotations during creation of a `DynamicDecouplingSequence` object

### CODE REVIEW

File: qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py [hunk 1]
```diff
@@ -95,6 +95,13 @@ def __init__(
             {"offsets": offsets, "duration": duration},
         )
 
+        rabi_rotations = np.asarray(rabi_rotations, dtype=np.float)
+        check_arguments(
+            np.all(rabi_rotations >= 0),
+            "rabi rotations must be non-negative. ",
```
Comment (charmasaur) [inline qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py:101]:
```suggestion
            "Rabi rotations must be non-negative.",
```

**Comment by ViktorPerunicic** [comment]
@tiborr, we'll fix the vault issue soon.

**Review by charmasaur — COMMENTED** [review]
Looks great, thanks! Made one little suggestion, aside from that we just need to sort out why the CI is failing (we'll look into that on the Q-CTRL side).

**Comment by tachikoma-li** [comment]
Shall we apply the conventional format of the PR title? (I can't remember what we decide to do with open sourced repo) @charmasaur @leoadec

**Comment by leoadec** [comment]
@tachikoma-li The text we added to https://elements.q-ctrl.com/code/contributing/#submitting-a-pull-request doesn't distinguish open source and private repos, so I think we're using the conventional commits format in all repos

**Review by charmasaur — APPROVED** [review]

**Pull request merged**